src/mds/schedule/copySetScheduler.cpp

```diff
@@ -108,12 +165,35 @@ int CopySetScheduler::DoCopySetSchedule(PoolIdType lid) {
                       << "on chunkserver:" << target
                       << " success. generator op: "
                       << op.OpToString() << "success";
+
+            oneRoundGenOp++;
```

Comment:
This piece of code coincides with the above

if (CopySetMigration(distribute, &op, &source, &target, &choose)) {
        // repeat
}

Reply:
most Most of the processing logic is different